<?xml version="1.0"?>
<sdf version="1.7">
  <model name="table">
    <link name="table_body">
      <visual name="table_visual">
        <geometry>
          <mesh>
            <uri>table_wide.obj</uri>
          </mesh>
        </geometry>
      </visual>
      <collision name="table_top">
        <pose> 0 0 -0.1 0 0 0</pose>
        <geometry>
          <box>
            <size>2.5 2.5 0.2</size>
          </box>
        </geometry>
      </collision>
    </link>
  </model>
</sdf>
